example-backend/src/main/resources/application-example-backend.yml

```
@@ -0,0 +1,166 @@
+server:
+  port: 8080
+  # this flag is needed to properly redirect to https:
+  forward-headers-strategy: framework
+  servlet:
+    context-path: /
+  error:
+    whitelabel:
+      enabled: true
+    include-stacktrace: never
+
+spring:
+  servlet:
+    multipart:
+      max-file-size: 10MB
+      max-request-size: 10MB
+  main:
+    banner-mode: off
+  jpa:
+    # Show SQL statements
+    show-sql: true
+    open-in-view: true
+    database-platform: org.hibernate.spatial.dialect.postgis.PostgisPG95Dialect
+    hibernate:
+      jdbc:
+        lob:
+          non_contextual_creation: true
+      ddl-auto: none
+    properties:
+      javax:
+        persistence:
+          sharedCache:
+            # Only entities with @Cacheable annotation will use L2 cache
+            mode: ENABLE_SELECTIVE
+      org:
+        hibernate:
+          # See https://docs.jboss.org/envers/docs/#configuration
+          envers:
+            default_schema: shogun_rev
+            audit_table_prefix: _rev
```

```suggestion
            audit_table_prefix: ''
            audit_table_suffix: _rev
```

this should be suffix instead of prefix, I think

see https://github.com/terrestris/shogun/pull/428

---

Nice catch @LukasLohoff!